requirements.txt

```diff
@@ -1,7 +1,18 @@
-matplotlib
+# Required for fairlearn
 numpy
 pandas
-pytest
 scipy
+
+# Required for environment
 flake8
-sklearn
\ No newline at end of file
+autopep8
+
+# Required for test
+# Pin pytest due to VS Code issue
+pytest==5.0.1
+scikit-learn
+
+# Required for notebooks
+jupyter
+matplotlib
```

Comment:
matplotlib is actually required for post processing if you run it in debug mode right now

Reply:
Anything else I should add in?

Reply:
I don't think anything's missing. pip install has no issues with comments this way, right?

Reply:
I've tested on a fresh environment :-)

Reply:
Are you planning on keeping the matplotlib dependency for postprocessing?

Reply:
I thought about this a little, and I think the plots are useful. Also, 99.9% of people who do machine learning will have matplotlib installed anyway. I don't think it's unreasonable. 

FYI I've made a note about discussing the package versioning with the entire group in the next meeting.